README.md

```diff
@@ -266,6 +266,7 @@ allowInvisibleElements | boolean | Whether to include elements that are not visi
 ignoreUnimportantViews | boolean | Enables or disables layout hierarchy compression. If compression is enabled, the layout hierarchy derived from the Acessibility framework will only contain nodes that are important for uiautomator testing. Any unnecessary surrounding layout nodes that make viewing and searching the hierarchy inefficient are removed. `true` by default
 elementResponseAttributes | string | Comma-separated list of element attribute names to be included into findElement response. By default only element UUID is present there, but it is also possible to add the following items: `name`, `text`, `rect`, `enabled`, `displayed`, `selected`, `attribute/<element_attribute_name>`. It is required that `shouldUseCompactResponses` setting is set to `false` in order for this one to apply.
 enableMultiWindows | boolean | Whether to include all windows that the user can interact with (for example an on-screen keyboard) while building the XML page source (`true`). By default it is `false` and only the single active application window is included to the page source.
+enableTopmostWindowFromActivePackage | boolean | Whether to get the window with the highest Z-order from the active package, regardless of whether it is focusable or not. By default it is `false` and only the active application window, implying that the window is focusable, is included to the page source.
```

Comment:
I would rather say: Whether to limit the window with the highest Z-order from the active package for interactions and page source retrieval

Reply:
Thanks for the suggestion. With 9b9e07ed21616d6f9601bcb22d665e913f734f55, I also rewrote the 2nd sentence too.